software-setup/software-installation.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Software setup instructions
+---
+
+**Table of contents**
+
+* TOC goes here
+{:toc}
+
+## RStudio Server
+
+You will **not** need to install R or RStudio on your computer.
+Instead, we have set up an RStudio Server for this workshop which you can access entirely via browser.
+Please refer to [these instructions](rstudio-login.md) for setting up your RStudio Server account.
+
+{% if site.workshop_type == "remote" %}
+
+## Zoom setup instructions
+
+We will use Zoom for all remote workshop sessions, as described in our [Workshop Logistics](../workshop/workshop-logistics.md) and [Zoom Procedures](zoom-procedures.md) pages.
+### Installing Zoom
+
+If you do not have Zoom installed yet, you will need to download the Desktop client from <https://zoom.us/download>.
+
+### Enabling screen share
+
+One way that we'll use Zoom is via screen-sharing; it is important that you are able to share your screen with us so we can help you, as needed in 1:1 consultations sessions or in breakout rooms.
+
+If you are working on a Mac, you will need to grant Zoom permissions to be able to use screen share, as follows:
+
+{% comment %}
+Zoom's instructions are not yet updated for Ventura, but we could write instead:
+
+Follow [this link](https://support.zoom.us/hc/en-us/articles/360016688031) for full instructions on how to grant Zoom these permissions, if you have not previously done so.
+{% endcomment %}
+
+#### For macOS Ventura and higher
+* Open "System Settings"
+* Navigate to "Privacy & Security"
+* Click "Screen Recording"
+* Ensure the Zoom app is toggled on
+  * If the Zoom app is not listed, click the plus sign to add Zoom to the list (you can find it in your Applications folder), and then toggle it on
+
+
+#### For macOS Monterey and lower
+* Open "System Preferences"
+* Navigate to "Security & Privacy"
+* Click "Privacy", and then select "Accessibility" from the sidebar.
+* Ensure the Zoom app is toggled on in _both sections_: "Allow the apps below to control your computer" and "Screen Recordings"
+  * If the Zoom app is not listed, click the plus sign to add Zoom to the list (you can find it in your Applications folder), and then toggle it on
+
+{% endif %}
+
+## Slack setup instructions
+
+We will use Slack to communicate with you both during and after the workshop, as described in our [Workshop Logistics](../workshop/workshop-logistics.md) and [Slack Procedures](slack-procedures.md) pages.
+
+
+### Installing Slack
+
+While Slack can be used directly from the browser at (<https://slack.com>), we recommend that you download Slack to be able to use its full features.
+Follow the download and installation instructions for your given operating system:
```

Comment:
Alternatively, we can just send everyone to `https://slack.com/downloads`, which will redirect to the correct download for each system.

Reply:
👆